packages/cli/src/config/extensions/update.ts

```diff
@@ -71,13 +73,33 @@ export async function updateExtension(
       workspaceDir: cwd,
       extensionEnablementManager,
     });
+
+    const extensionStorage = new ExtensionStorage(extension.name);
```

Comment:
All this logic should probably exist inside `installOrUpdateExtension`, that function should fully handle safely updating (or installing) extensions without needing to do other logic before/after calling it